src/Razor/src/Microsoft.AspNetCore.Razor.Language/Legacy/CSharpCodeParser.cs

```diff
@@ -482,7 +482,14 @@ private CSharpStatementBodySyntax ParseStatementBody(Block block = null)
                 // Set up auto-complete and parse the code block
                 var editHandler = new AutoCompleteEditHandler(Language.TokenizeString);
                 SpanContext.EditHandler = editHandler;
-                ParseCodeBlock(builder, block, acceptTerminatingBrace: false);
+                ParseCodeBlock(builder, block);
```

Comment:
I assume from this that `ParseCodeBlock` was simplified so it can be reused?

Was `acceptTerminatingBrace` unused?